### PR TITLE
The first version of the Marketplace chapter #28

### PR DESCRIPTION
**Pull request by irinaponomarevahaulmont**
Please review.

### CODE REVIEW

File: content/modules/studio/pages/marketplace.adoc [hunk 1]
```diff
@@ -1,3 +1,57 @@
 = Add-ons Marketplace
 
-TODO
+Studio provides useful UI to manage add-ons in your application. The Add-ons dialog is accessible from the Editor Toolbar.
```
Comment (alexbudarov) [inline content/modules/studio/pages/marketplace.adoc:3]: > from the Editor Toolbar

It's not "Editor" Toolbar. Editor Toolbar is a bar on top of the source code editor.

File: content/modules/studio/pages/marketplace.adoc [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+== Add-on Installation
+You can install an add-on by clicking the *_Install_* button. During the installation changes are made to the project's configuration files such as adding the necessary libraries to `pom.xml`, `package.json`.
+
+Studio will suggest adding new queries, mutations and types that come with the Add-on to the project's schema.
+
+image::marketplace/add-on-preview.png[align=center]
+
+Click *_OK_* to complete the installation and wait until the installation process is over. A notification will be displayed at the bottom of the editor window.
+
+image::marketplace/installed-add-on-message.png[align=center]
```
Comment (alexbudarov) [inline content/modules/studio/pages/marketplace.adoc:28]: On the screenshot, an **un**installed notification is displayed.

File: content/modules/studio/pages/marketplace.adoc [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+[[installed-tab]]
+== The Installed Tab
+
+The *_Installed_* tab displays add-ons that are already added to the current project.
+
+image::marketplace/installed-add-on.png[align=center]
+
+[[uninstalling-add-on]]
+== Uninstalling Add-ons
+To uninstall an add-on, you need to go to the `Installed`
```
Comment (alexbudarov) [inline content/modules/studio/pages/marketplace.adoc:48]: Wrong styling for "Installed" and "Uninstall"
(these are UI captions)

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Corrections made, please review.

**Pull request merged**